packages/mask/src/components/InjectedComponents/ProfileTabContent.tsx

```diff
@@ -340,7 +340,7 @@ function Content(props: ProfileTabContentProps) {
                 <div className={classes.root}>
                     <PluginCardFrameMini>
                         <GrantPermissions
-                            permissions={lackPluginDefine?.enableRequirement.host_permissions ?? []}
+                            permissions={lackPluginDefine?.enableRequirement.host_permissions ?? EMPTY_LIST}
```

Comment:
do we need some kind of linter to do this? or it will cause many false positive?